tools/cli/help.txt

```diff
@@ -73,6 +73,7 @@ Options:
                    uses port N+1 and a port specified by --app-port.
                    Specify as --port=host:port to bind to a specific
                    interface.
+  --open, -o       Opens a browser window when the app starts.
```

Comment:
Should we also document this option in https://github.com/meteor/meteor/blob/devel/docs/source/commandline.md ?

Reply:
Yes, this needs to be documented. And I would like to lobby to make it the default in 3.0.